Source/Tests/Paths/TempDirPathTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace Bloodmasters.Tests.Paths;
```

Comment:
I bet this file includes BOM. Let's figure out what creates it in your environment :)

What IDE/editor do you use? And how are you creating the files?

Reply:
I'm using Rider. And i found corresponding setting:

![image](https://github.com/ForNeVeR/bloodmasters/assets/38229504/96ba6cc3-f50f-453f-b0b4-74d3cb6ad35b)

I didn't touch it, that is a default value

Reply:
Changed it to always create without BOM

Reply:
Ok, set it up to no BOM. And this is stupid. Vote for [IDEA-257119](https://youtrack.jetbrains.com/issue/IDEA-257119).